tests/test_singleton.py

```diff
@@ -0,0 +1,13 @@
+from unittest import TestCase
+from electionguard.singleton import Singleton
+
+
+class TestSingleton(TestCase):
```

Comment:
Added singleton test to raise test coverage since this file was particularly low.